torchgeo/datasets/chesapeake.py

```diff
@@ -540,6 +581,34 @@ def __init__(
 
         super().__init__(transforms)
 
+        lc_colors = []
+        for i in range(min(self.lc_cmap.keys()), max(self.lc_cmap.keys()) + 1):
+            if i in self.lc_cmap:
+                lc_colors.append(
+                    (
+                        self.lc_cmap[i][0] / 255.0,
+                        self.lc_cmap[i][1] / 255.0,
+                        self.lc_cmap[i][2] / 255.0,
+                    )
+                )
+            else:
+                lc_colors.append((0, 0, 0))
```

Comment:
I think this could be done without for-loops. Maybe something like:
```python
lc_colors = np.zeros((max(lc_cmap.keys()) + 1, 4))
lc_colors[list(lc_cmap.keys())] = list(lc_cmap.values())
lc_colors[:, :2] /= 255
```

Reply:
Nice!